### ISSUE TITLE
Customer Support Tracking - Dashboard 

### ISSUE THREAD

**Issue by jchasia-adhoc** [department-of-veterans-affairs/va.gov-team#17356]
## Problem Statement 
Without a better understanding of how VSP teams' time is spent on customer support work, it is difficult to identify areas to focus on to make improvements that enable VFS teams to find answers or get unblocked on their own.

How might we make customer support data available to VSP stakeholders so we can identify areas with high burdens of manual support and track the volume of support by team and practice area over time?

## Hypothesis or Bet
By collecting data on Slack customer support requests and making it easy the generate reports, we expect to identify “problem areas” with high burdens of manual support to be prioritized in future VSP initiatives.

## We will know we're done when... ("Definition of Done")
When we can generate reports on the following metrics:
- [ ] Total Slack customer support requests - by VFS team, practice area
- [ ] Request type: no documentation, category
- [ ] VSP time spent: per request, category, week, practice area

## Known Blockers/Dependencies 
- VSP Engineering Leadership designating a BE developer to help us maintain the database and iterate on the Support Bot.
- Planning to work with the Analytics Team to pull data generated by the Support Bot into BigQuery.
- If it turns out BigQuery is not an option in Q1, the backup plan is to use Grafana to generate customer support reports with the data archived in Loki or Datadog. This work would be dependent on the BE developer referenced in the first bullet point.

## Projected Launch Date
February 2021 - assuming the resource referenced above is available at the beginning of Q1.

### ... tested?

- [ ] Usability test (_TODO: link_) has been performed, to validate that new changes enable users to do what was intended and that these changes don't worsen quality elsewhere. If usability test isn't relevant for this change, document the reason for skipping it.
    - [ ] ... and issues discovered in usability testing have been addressed.
    * _Note on skipping: metrics that show the impact of before/after can be a substitute for usability testing._
- [ ] End-to-end [manual QA](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/README.md) or [UAT](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/planning/what-is-uat.md) is complete, to validate there are no high-severity issues before launching
- [ ] _(if applicable)_ New functionality has thorough, automated tests running in CI/CD

### ... documented?

- [ ] New documentation is written pursuant to our [documentation style guide](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/documentation/style-guide)
- [ ] Product is included in the [List of VSP Products](https://docs.google.com/spreadsheets/d/1Fn2lD419WE3sTZJtN2Ensrjqaz0jH3WvLaBtn812Wjo/edit#gid=0)
    * _List the existing product that this initiative fits within, or add a new product to this list._
- [ ] Internal-facing: there's a [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsp/product-management/product-outline-template.md) checked into [`products/platform/PRODUCT_NAME/`](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/platform/)
    * _Note: the Product Directory Name should match 1:1 with the List of VSP Products_
- [ ] External-facing: a [VFS-facing README](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsp/product-management/product-readme-template.md) exists for this product/feature tool
    - [ ] ... and should be located at `platform/PRODUCT_NAME/README.md`
- [ ] External-facing: a [User Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsp/product-management/writing-user-guides.md) exists for this product/feature/tool, and is updated for changes from this initiative
    - [ ] ... and should be linked from the VFS-facing README for your product
    - [ ] ... and should be located within `platform/PRODUCT_NAME/`, unless you already have another location for it
    - [ ] _(if applicable)_... and post to [#vsp-content-ia](https://dsva.slack.com/channels/vsp-content-ia) about whether this should be added to the [Documentation homepage](https://department-of-veterans-affairs.github.io/va.gov-team/)
- [ ] _(if applicable)_ Post to [#vsp-service-design](https://dsva.slack.com/channels/vsp-service-design) for external communication about this change (e.g. VSP Newsletter, customer-facing meetings)

### ... measurable

- [ ] _(if applicable)_ This change has clearly-defined success metrics, with instrumentation of those analytics where possible, or a reason documented for skipping it.
    * For help, see: [Analytics team](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/analytics)
- [ ] This change has an accompanying [VSP Initiative Release Plan](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new/choose).

### Documentation
* **`PRODUCT_NAME`**: _directory name used for your product documentation_
* **Product Outline**: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/platform/customer-support-tracking/product-outline.md
* **README**: _link to VFS-facing README for your product_
* **User Guide**: _link to User Guide_

### Testing

* **Usability test**: _link to GitHub issue, or provide reason for skipping_
* **Manual QA**: _link to GitHub issue or documented results_
* **Automated tests**: _link to tests, or "N/A"_

### Measurement

* **Success metrics**: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/platform/customer-support-tracking/product-outline.md
* **Release plan**: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.github/ISSUE_TEMPLATE/vsp-release-plan.md

## TODOs
- [x] Convert this issue to an epic
- [x] Add your team's label to this epic

**Comment by Khyatinp18** [comment]
2/21/2022:

Closing this issue since it's no longer needed.
